File: leet_code/math/204/204.dart
int countPrime(int n) {
  if (n <= 2) {
    return 0;
  }
  List<bool> isPrime = List.filled(n, true);
  isPrime[0] = false;
  isPrime[1] = false;
  for (int i = 2; i * i < n; i++) {
    if (isPrime[i]) {
      for (int j = i * i; j < n; j += i) {
        isPrime[j] = false;
      }
    }
  }
  int count = 0;
  for (int i = 0; i < n; i++) {
    if (isPrime[i]) {
      count++;
    }
  }
  return count;
}

void main() {
  print(countPrime(10));
}

File: leet_code/math/204/204.py
def countPrimes(n)->int:
    def is_prime(num)->bool:
        if num<=1:
            return False
        for i in range(2,num):
            if num%i==0:
                return False
        return True
    count = 0
    for i in range(0,n):
        if is_prime(i):
            count+=1
    return count
print(countPrimes(10))